app/components/app_patient_medical_history_card_component.html.erb

```diff
@@ -0,0 +1,48 @@
+<div class="nhsuk-card" id="consent">
+  <div class="nhsuk-card__content">
+    <h2 class="nhsuk-card__heading nhsuk-heading-m">
+      Medical history
+    </h2>
+
+    <% if @consent_response.triage_needed? %>
+      <% if triage_present? %>
+        <p>Triage needed</p>
+
+        <ul class="njsuk-list njsuk-list--bullet">
```

Comment:
```suggestion
        <ul class="nhsuk-list nhsuk-list--bullet">
```

Reply:
I know I'm still struggling occasionally with this kbd, but ... did I really type those out by hand?? 🤦